src/main/java/org/jitsi/videobridge/health/Health.java

```diff
@@ -121,7 +121,7 @@ private static void check(ColibriShim.ConferenceShim conferenceShim)
             }
 
             // Create and install the transport manager
-            conferenceShim.getOrCreateChannelBundle(endpoint.getID());
+            conference.getShim().getOrCreateChannelBundle(endpoint.getID());
```

Comment:
since we hide the shim for `getOrCreateEndpoint`, should we hide it here as well? that is, just call `conference.getOrCreateChannelBundle` here?

Reply:
I think `getOrCreateEndpoint` should be in `Conference` because it's "native" (deals with `Endpoint`s that are part of `Conference`), while `getOrCreateChannelBundle` deals with channel bundles that are shim-only. I'd push everything that is shim-only to the shims.

Reply:
i considered TransportManagers the 'native' side of a channel bundle, is that not accurate?

Reply:
I think as them as part of an `Endpoint`

Reply:
I agree there's more work there -- actually moving the transport manager to `Endpoint`, but I'd leave that for a later PR.